examples/pagination_submitquery.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2022 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+import logging
+import json
+import requests
+import sys
+
+# search engine url
+base_url = "http://127.0.0.1:5577/api/v1/"
+submit_query_url = "http://127.0.0.1:5577/api/v1/resources/submitquery"  # noqa
```

Comment:
`base_url` isn't used.
I tried this:
```
base_url = "https://idr-testing.openmicroscopy.org/searchengine/api/v1/"
submit_query_url = base_url + "resources/submitquery"  # noqa
```
But this isn't working for me..

`resp = requests.post(url, data=data)`
below is giving me a 404, although the URL appears correct because if I do this with the same URL
`resp = requests.get(url)`
I get `405 Method Not Allowed`.

NB: I had to comment-out the try/except block below to be able to see what's going on here, since it was actually failing on `returned_results = json.loads(resp.text)`

Reply:
Could you please add `/` by the end and try again, i.e.

```
base_url = "https://idr-testing.openmicroscopy.org/searchengine/api/v1/"
submit_query_url = base_url + "resources/submitquery/" 
```
I have updated the example

Reply:
Looks good, thanks - I guess GET must be less picky about the extra `/` than POST!